src/index.html

```diff
@@ -1,13 +1,373 @@
+          <h1 class="header__title">ХУДОЖНІЙ МУЗЕЙ</h1>
+          <div class="header__tickets">
+            <a href="#" class="tikets">
+              <span class="tikets__text">КВИТКИ</span>
+              <div class="tikets__button button button--grean"></div>
+            </a>
+          </div>
+          <a href="#current-events" class="header__events">ПОДІЇ</a>
+
+        </div>
+      </div>
+    </header>
+
+    <aside class="body__aside aside" id="aside">
+      <div class="aside__wrapper">
+        <div class="aside__container">
+          <div class="aside__top">
+            <a href="#" class="aside__icon icon"></a>
+
+            <nav class="aside__nav nav-aside">
+              <ul class="nav-aside__list">
+                <li class="nav-aside__item">
+                  <a href="#gallery" class="nav-aside__link">Галерея</a>
+                </li>
+                <li class="nav-aside__item">
+                  <a href="#" class="nav-aside__link">Про Нас</a>
+                </li>
+                <li class="nav-aside__item">
+                  <a href="#current-events" class="nav-aside__link">Виставки та події</a>
+                </li>
+                <li class="nav-aside__item">
+                  <a href="#contacts" class="nav-aside__link">Контакти</a>
+                </li>
+              </ul>
+            </nav>
+          </div>
+
+          <div class="aside__line"></div>
+
+          <div class="aside__bottom">
+
+            <div class="aside__schedule schedule schedule--asid">
+              <h4 class="schedule__title">Графік роботи</h4>
+
+              <ul class="schedule__list schedule__list--asid">
+                <li class="schedule__item">
+                  12:00 — 20:00 <span class="schedule__text">середа</span>
+                </li>
+                <li class="schedule__item">
+                  12:00 — 20:00 <span class="schedule__text">четвер</span>
+                </li>
+                <li class="schedule__item">
+                  12:00 — 20:00 <span class="schedule__text">п’ятниця</span>
+                </li>
+                <li class="schedule__item">
+                  11:00 — 19:00 <span class="schedule__text">субота</span>
+                </li>
+                <li class="schedule__item">
+                  11:00 — 19:00 <span class="schedule__text">неділя</span>
+                </li>
+              </ul>
+
+              <p class="schedule__atantion schedule__atantion--asid">
+                Вихідні: <span class="schedule__text">понеділок, вівторок Каса припиняє роботу за 1 годину до закриття музею</span>
+              </p>
+            </div>
+
+          </div>
+        </div>
+      </div>
+      <a href="#" class="aside__close"></a>
+    </aside>
+
+    <main class="main body__main">
+      <section class="main__section current-events" id="current-events">
+        <div class="current-events__container">
+          <h1 class="section-titile">Актуальні події</h1>
```

Comment:
You need to use another level header here, because usually this tag is used only 1 time and in one place on the page
```suggestion
          <h2 class="section-titile">Актуальні події</h2>
```